renovate.json5

```diff
@@ -17,6 +17,13 @@
       "matchPackageNames": ["androidx.browser:browser"],
       "matchUpdateTypes": ["major", "minor"],
       "enabled": false
+    },
+    {
+      // somehow renovate gets confused by the android property in gradle.properties,
+      // so let's just exclude it and hopefully clean up the dashboard
+      "matchPackageNames": ["string:rum.version"],
+      "matchUpdateTypes": ["major", "minor"],
```

Comment:
don't need this if blanket disabling
```suggestion
```

Reply:
Yeah clearly this is all very new to me. :) Thank you.